Clubs_level_1_char_sheet.html

```diff
@@ -0,0 +1,351 @@
+
+<html>
+
```

Comment:
Why is this in the templates folder? I've been using that for code that is the template, not examples. Is it an input? is it an example output?

Reply:
This is an example Output, not sure where that should go. Happy to move or adjust accordingly (can remove entirely) depending on how we want structure of this to go.

Reply:
I can work on a proposal for how to organize this kind of export